src/protonuke/protonuke.go

```diff
@@ -37,6 +37,7 @@ var (
 	f_httpImageSize = flag.Int("httpimagesize", 3, "size of image, in megabytes, to serve in http/https pages")
 	f_httpTLSCert   = flag.String("httptlscert", "", "file containing public certificate for TLS")
 	f_httpTLSKey    = flag.String("httptlskey", "", "file containing private key for TLS")
+	f_tlsVersion		= flag.String("tlsversion", "", "Select a TLS version for the client: tls1.0, tls1.1, tls1.2")
```

Comment:
Update help text and bracket options:

   Select a maximum client TLS version [1.0, 1.1, 1.2]